src/Components/CookieConsent.js

```diff
@@ -22,10 +22,13 @@ function CookieConsent() {
   return (
     <ReactCookieConsent
       buttonText="accept"
+      enableDeclineButton
+      declineButtonText="decline"
```

Comment:
I would also use the option `flipButtons` to make the accept button before the decline button. Than `mr-2` needs to move from `declineButtonClasses` to `buttonClasses`.